homeassistant/helpers/selector.py

```diff
@@ -125,6 +127,8 @@ class DeviceSelector(Selector):
             vol.Optional("model"): str,
             # Device has to contain entities matching this selector
             vol.Optional("entity"): EntitySelector.CONFIG_SCHEMA,
```

Comment:
This selector is changing. Please inline the old config schema. 

CC @emontnemery